apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.html.ts

```diff
@@ -54,7 +72,22 @@ export function RequestCourse({
         })}
         <main id="content" class="container">
           <h1 class="sr-only">Request a Course</h1>
-          ${CourseRequestsCard({ rows })}
+          ${CourseRequestsCard({ rows })} ${EncodedData(lti13Info, 'courseRequestLti13Info')}
+          ${Modal({
+            id: 'lti13FillModal',
+            title: html`Auto-fill with ${lti13Info?.['cr-institution'] ?? 'LMS'} data?`.toString(),
+            body: html`<p>
+                You appear to be coming from a course in another learning system. Should we
+                partially fill in this request form with information from that course?
+              </p>
+              <p>(You can edit it after it's auto-filled.)</p>`,
```

Comment:
Plus or minus some Prettier formatting:

```suggestion
            body: html`
              <p>
                You appear to be coming from a course in another learning system. Should we
                partially fill in this request form with information from that course?
              </p>
              <p>(You can edit it after it's auto-filled.)</p>
              `,
```

Reply:
Fixed in 0a22979fe